gtfs/retriever/gtfs_retrieve.py

```diff
@@ -42,7 +42,7 @@ def get_local_date_and_time_hyphen_delimited():
 
 
 def ftp_get_file(local_path, host=MOT_FTP, remote_path=GTFS_FILE_NAME):
-    """ get file remote_name from FTP host host and copy it into local_path"""
+    """ get file remote_path from FTP host and copy it into local_path"""
     print("Starting to download '%s' from host '%s' => '%s'" % (remote_path, host, local_path))
```

Comment:
use python logging instead of prints

Reply:
this needs proper logging, because we upload to s3 only when there is a diff. without proper logging we won't be able to tell whether there was no change in MoT's FTP or an issue on our end.